### DISCUSSION TITLE
Good first Issue

### DISCUSSION THREAD

**Discussion by DoTheBestToGetTheBest** [paradigmxyz/reth#4775] [Help]
Hello, could we have some good first issue for beginner in Ethereum ? I'am really interested and it's one of my dream to be ethereum developer. Please and thank you

**Comment by onbjerg** [comment]
Hey! Check out these https://github.com/paradigmxyz/reth/labels/D-good-first-issue

**Reply by DoTheBestToGetTheBest** [reply]
Hey, thank you for your answer

most of them are already taken or there are complex problems to resolve this. I'm looking for something friendlier

**Comment by DoTheBestToGetTheBest** [comment]
Guys can we have more good first issue Please?  I really wants to improve my knowledge and skill in this project

**Comment by DoTheBestToGetTheBest** [comment]
@mattsse hello please Mat, open new good issue please and i'm really sorry to ping. You don't imagine how i want to be better in reth developement
